<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1297</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>31</y>
      <width>501</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="schoolLogoLabel">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>10</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>181</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="calculateButton">
       <property name="text">
        <string>Oblicz średnią</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addStudentButton">
       <property name="text">
        <string>Dodaj ucznia</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
